.github/workflows/release_packages.yml

```diff
@@ -1,4 +1,4 @@
-name: Pre-release ui-components
+name: Pre-release packages
```

Comment:
We only every wanted to pre-release ui-components - a commit per commit pre-release was never intended for our other packages. I think the goal was to pre-release just uic on commit to dev, and fully release everything on commits to main (when we had both branches). I don't know that we need to publish any of these as npm packages?

Reply:
Good to know @emilyjablonski! I also don't personally see the need for publishing these as npm packages. In that case, are there any potential side effects to removing this release process altogether and just updating our imports to the local files?

Reply:
Not that I can think of, but it's possible we use versioning when we deploy the backend, let's check in w the team - @ludtkemorgan any thoughts here?

Reply:
I don't have any issues with us not publishing to NPM. However, one thing we are getting from lerna is having the changelog get updated. With the full removal of these steps that will no longer happen. And also as a thing to note for both public and partners site it will always be set to this version going forward `"@bloom-housing/backend-core": "^7.6.0"`. That isn't an issue, but I'm wondering if lerna is even necessary anymore at that point.

Reply:
If we want automated changelog updates we can version and update the changelog without publishing to npm if that's a preferred flow